util/process_wrapper/flags.rs

```diff
@@ -0,0 +1,304 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::{HashMap, HashSet};
+use std::error::Error;
+use std::fmt;
+use std::fmt::Write;
+use std::mem::{replace, take};
+
+#[derive(Debug, Clone)]
+pub(crate) enum FlagParseError {
+    UnkownFlag(String),
+    RequiredArgumentMissing(String),
```

Comment:
It seems we don't have required arguments, should we delete this and the code related to required args?

Reply:
Done.